### ISSUE TITLE
graphql operation with parameter doesn't work

### ISSUE THREAD

**Issue by AnsonCode** [wundergraph/wunderbase#1]
when i use graphql operation with parameter,it doesn't work,like this:

![image](https://user-images.githubusercontent.com/20960032/190584717-ac9f9686-a170-408b-b42a-253b68c38fa4.png)

but if don't use paramter,it works,like this:

![image](https://user-images.githubusercontent.com/20960032/190584885-9aeeeeed-a7e7-43e3-b4ee-6bcb2cf4c808.png)

**Comment by jensneuse** [comment]
That's correct! Prisma doesn't support variable definitions.
That's why we've built something into [WunderGraph](https://github.com/wundergraph/wundergraph) to "inline" variable definitions.

**Comment by AnsonCode** [comment]
Yes, I'v submit an issue [here](https://github.com/prisma/prisma-engines/issues/3195),but be  refused.

**Comment by AnsonCode** [comment]
> That's correct! Prisma doesn't support variable definitions. That's why we've built something into [WunderGraph](https://github.com/wundergraph/wundergraph) to "inline" variable definitions.

I think we should migrate this function for wunderbase.

**Comment by jensneuse** [comment]
As we're only always using wunderbase behind WunderGraph, I don't see much value in doing so. It's fast, but not inexpensive, and shouldn't run twice in the request chain.
